lib/Qi/Model/Base.pm

```diff
@@ -0,0 +1,35 @@
+package Qi::Model::Base;
+use strict;
+use warnings;
+use Mojo::Base 'Mojolicious';
+use SQL::Abstract;
+
+#### self Methods ####
+has 'sql'; # $self->sql  for SQL::Abstract
```

Comment:
This is where you want to define the SQL::Abstract object.  `has sql => sub { state $sql = SQL::Abstract->new };`